packages/core/src/components/Spinner/Spinner.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * The `Spinner` component
```

Comment:
This is redundant. The heading above it in the generated documentation essentially says the same thing

![image](https://user-images.githubusercontent.com/371943/29836485-6357689e-8cc3-11e7-8f5f-9559c02c618d.png)